airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -24608,10 +24608,10 @@
               "type": "object",
               "required": [ "auth_type", "api_key" ],
               "properties": {
-                "credentials_title": {
+                "auth_type": {
                   "type": "string",
-                  "const": "API Key",
```

Comment:
Please keep the original constant as it's more human readable

Reply:
fixed